ops/static/css/fonts.css

```diff
@@ -4,47 +4,47 @@
   font-family: 'Raleway';
   font-style: normal;
   font-weight: 300;
-  src: local('Raleway Light'), local('Raleway-Light'), url(http://fonts.gstatic.com/s/raleway/v11/ZKwULyCG95tk6mOqHQfRBCEAvth_LlrfE80CYdSH47w.woff2) format('woff2');
+  src: local('Raleway Light'), local('Raleway-Light'), url(//fonts.gstatic.com/s/raleway/v11/ZKwULyCG95tk6mOqHQfRBCEAvth_LlrfE80CYdSH47w.woff2) format('woff2');
```

Comment:
That said, perhaps that's something you can address in a follow-up pull request. By the way, try not to make your changes on the `master` branch. When you're about to make changes, create a new feature branch first as noted on our [Git Tips](https://github.com/getpelican/pelican/wiki/Git-Tips) page. Then submit your pull request from that branch.